Pods/Local Podspecs/DarklyEventSource.podspec.json

```diff
@@ -9,15 +9,15 @@
   },
   "source": {
     "git": "https://github.com/launchdarkly/ios-eventsource.git",
-    "tag": "3.1.0"
+    "tag": "v3.1.1"
```

Comment:
This is probably not right -- we don't prefix tags with `v`.

Reply:
I saw this too, but it's an artifact of CocoaPods. The LDEventSource.podspec doesn't have the 'v' in the tag.

I tried pod install using 3.1.1, followed by pod update to return to 3.1.2. That didn't change this.

If you want me to track this down I will, but I suspect it came from releasing 3.1.1 with the `v` in the tag.